ra/ra.go

```diff
@@ -358,6 +360,34 @@ func (ra *RegistrationAuthorityImpl) checkPendingAuthorizationLimit(ctx context.
 	return nil
 }
 
+func (ra *RegistrationAuthorityImpl) checkInvalidAuthorizationLimit(ctx context.Context, regID int64) error {
+	limit := ra.rlPolicies.InvalidAuthorizationsPerAccount()
+	saGRPC, ok := ra.SA.(bgrpc.StorageAuthorityClientWrapper)
+	if limit.Enabled() && ok {
```

Comment:
Instead of wrapping this entire block in this statement could you do `if !(limit.Enabled() &&ok) { return nil }`?